tensorflow_serving/tools/pip_package/setup.py

```diff
@@ -44,7 +44,7 @@
   sys.argv.remove('--project_name')
   sys.argv.pop(project_name_idx)
 
-_TF_REQ = ['tensorflow'+_TF_VERSION_SANITIZED]
+_TF_REQ = ['tensorflow-cpu'+_TF_VERSION_SANITIZED]
 
```

Comment:
this will break for people who have `tensorflow` package installed:

https://pypi.org/project/tensorflow/

is there a way to specify a list of packages, where any one of them can be installed to satisfy the requirement?

Reply:
How about this?

https://github.com/tensorflow/serving/pull/1847/commits/f183188d7d555e88e111c1360d135f2490e6ac85

```
import pkg_resources

pkg_released = set(["tensorflow", "tensorflow-cpu"])
pkg_installed = set(map(lambda pkg: pkg.project_name, pkg_resources.working_set))
tensorflow_installed = list(pkg_released & pkg_installed) + ["tensorflow"]

_TF_REQ = [tensorflow_installed[0]+_TF_VERSION_SANITIZED]
```